plugins/pagerduty/README.md

```diff
@@ -0,0 +1,88 @@
+# PagerDuty
+
+## Overview
+
+This plugin displays PagerDuty information about an entity such as if there are any active incidents and what the escalation policy is.
+
+There is also an easy way to trigger an alarm directly to the person who is currently on-call.
+
+This plugin requires that entities are annotated with an [integration key](https://support.pagerduty.com/docs/services-and-integrations#add-integrations-to-an-existing-service). See more further down in this document.
+
+This plugin provides:
+
+- A list of incidents
+- A way to trigger an alarm to the person on-call
+- Information details about the person on-call
+
+## Setup instructions
+
+Install the plugin:
+
+```bash
+yarn add @backstage/plugin-pagerduty
+```
+
+Add it to the app in `plugins.ts`:
+
+```ts
+export { plugin as Pagerduty } from '@backstage/plugin-pagerduty';
+```
+
+Add it to the `EntityPage.ts`:
+
+```ts
+import {
+  isPluginApplicableToEntity as isPagerDutyAvailable,
+  PagerDutyCard,
+} from '@backstage/plugin-pagerduty';
+// add to code
+{
+  isPagerDutyAvailable(entity) && (
+    <Grid item md={6}>
+      <PagerDutyCard entity={entity} />
+    </Grid>
+  );
+}
+```
+
+## Client configuration
+
+The client takes a config object which contains an instance of the `discoveryApi` which is used to look up the `baseUrl` of the `proxy`, and a `eventUrl` which is defaulted to: "https://events.pagerduty.com/v2".
+
+In `apis.ts`:
+
+```ts
+createApiFactory({
```

Comment:
Are these instructions not obsolete now? If now, maybe making it a little move obvious this is optional would be great.

Reply:
Yeah, it's probably useful to mention that "if you want to customize or replace the API implementation, ..."